website/content/docs/enterprise/long-term-support.mdx

```diff
@@ -0,0 +1,121 @@
+---
+layout: docs
+page_title: Long Term Support (LTS)
+description: >-
+  Consul Enterprise offers a Long Term Support (LTS) release program. Learn about benefits and terms, including which versions receive extended support, for how long, and fix eligibility.
+---
+
+# Long Term Support (LTS)
+
+<EnterpriseAlert>
+  This program requires Consul Enterprise.
+</EnterpriseAlert>
+
+Consul Enterprise offers annual Long Term Support (LTS) releases starting with version v1.15. LTS releases maintain longer fix eligibility and support larger upgrade jumps.
+
+This page describes the LTS program, its benefits for Enterprise users, and the support the program includes.
+
+## Overview
+
+Many Enterprise organizations want to minimize their operational risk by operating a maintained Consul version that is eligible for critical fixes and does not require frequent upgrades.
+
+Annual Consul Enterprise LTS releases enable organizations to receive critical fixes without upgrading their major version more than once per year.
+
+The following table lists the lifecycle for each existing LTS release, including expected end of maintenance (EOM).
+
+| LTS release | Release date  | Estimated EOM date                |
+| ----------- | ------------- | --------------------------------- |
+| v1.18 Ent   | Pending       | Feb 28, 2026 (v1.24 Ent release)  |
+| v1.15 Ent   | Feb 23, 2023  | Feb 28, 2025 (v1.21 Ent release)  |
+
+## Release lifecycle
+
+Consul Enterprise LTS releases are maintained for longer than other Consul releases,
+as described in the following sections.
+
+### Standard Term Support lifecycle
+
+All major releases of Consul Enterprise and community edition receive _Standard Term Support_ (STS)
+for approximately one year, per HashiCorp's
+[Support Period Policy](https://support.hashicorp.com/hc/en-us/articles/360021185113-Support-Period-and-End-of-Life-EOL-Policy).
+
+With STS, each major release branch is maintained until it is
+three (3) releases from the latest major release.
+For example, Consul v1.14.x is maintained until the release of Consul v1.17.0.
+
+### Long Term Support lifecycle
+
+Starting with Consul Enterprise v1.15, the first major release of the calendar year
+receives _Long Term Support_ (LTS) for approximately 2 years.
+The first major release of the calendar year typically occurs in late February.
+
+An LTS release is maintained until it is six (6) releases from the latest major release.
+For example, Consul Enterprise v1.15.x is maintained until the release of Consul Enterprise v1.21.0.
+
+During the LTS window, [eligible LTS fixes](#fix-eligibility) are provided through
+a new minor release on the affected LTS release branch.
+
+[![Consul Enterprise Long Term Support lifecycle diagram](/img/long-term-support/consul-enterprise-long-term-support-lifecycle.png)](/img/long-term-support/consul-enterprise-long-term-support-lifecycle.png)

```

Comment:
The intent of this link format was to make the image itself clickable (to open the original file).

I think I did that for a different image in a previous PR that looked OK at 2048 px width, but it could be helpful to open the original SVG at a bigger size.

That said, I don't think that's necessary here.

I'll look at another page that uses dark mode images to understand the pattern for how that's done.

Reply:
Okay, found the pattern:
```
![Diagram that provides an overview of the Consul Architecture on ECS](/img/ecs/consul-on-ecs-architecture-dataplanes.png#light-theme-only)
![Diagram that provides an overview of the Consul Architecture on ECS ](/img/ecs/consul-on-ecs-architecture-dataplanes-dark.png#dark-theme-only)
```